libs/core/output.ts

```diff
@@ -222,7 +236,7 @@ namespace motors {
             // if we've recently completed a motor command with brake
             // allow 500ms for robot to settle
             if (this._brake)
-                pause(500);
```

Comment:
Now the default is 10? `this._settleTime` set 10 in constructor

Reply:
500 ms is too long, because when program has 10 movements it will wait 5 seconds
It also related to #875

Reply:
> Why is the default is 10? `this._settleTime` set 10 in constructor

@abchatra This number needs to be changed! It creates problems if you need to participate in competitions. If there is a need to look at all the steps how the robot executes commands, then it is really possible to change the number by a special function (which was added here) so that the robot waits...

We propose these changes for a reason. In block mode, this may also be needed by children who are still small for javasctipt, but are already participating in competitions.

Reply:
I am ok with the change as long as we understand 10ms is the right number and not 20 or 50ms. Let us know the testing you have done. 

However I don't understand the need for user to change this number. If 10ms is the right number why does user need to change it. What is the scenario?

I am not an expert in EV3 usage and any input on your end is much appreciated.

Reply:
@pelikhan

Reply:
> I am ok with the change as long as we understand 10ms is the right number and not 20 or 50ms. Let us know the testing you have done.
> 
> However I don't understand the need for user to change this number. If 10ms is the right number why does user need to change it. What is the scenario?
> 
> I am not an expert in EV3 usage and any input on your end is much appreciated.

This number gives a delay after the robot stops. I increased to 1000 and the robot stood after stopping for 1 second. I set less, 10 ms, now there is no delay. I did not try to set 0. But if you want, I will try it separately.

Reply:
Why does user need to change this using settleTIme API? Can we just change the default to 10ms?

Reply:
a.k.a simple one line change pause(10);

Reply:
@abchatra In the first version, I proposed so. But @maximmasterr I saw the need to use the method. Use cases may be, but I think there is no great need for this. Probably I would not use this method and it suited me 10 ms as standard.

Reply:
Adding setSettle api will give users ability to set settle time some ribots has big inertia and they need more settle time or our robot dont need settle time at all we set it to zero

Reply:
@pelikhan to pitch in his opinion. I prefer just updating the settle time to 10ms if that works for most of the use cases.